test/framework/src/test/java/org/elasticsearch/test/AbstractXContentTestCaseTests.java

```diff
@@ -0,0 +1,52 @@
+package org.elasticsearch.test;
+
+import org.elasticsearch.common.bytes.BytesReference;
+import org.elasticsearch.common.xcontent.ToXContent;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+import org.elasticsearch.common.xcontent.XContentParser;
+import org.elasticsearch.common.xcontent.XContentType;
+
+import java.io.IOException;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+
+public class AbstractXContentTestCaseTests extends ESTestCase {
+
+    private static final int NUMBER_OF_TEST_RUNS = 10;
+
+    public void testSomethign() throws IOException {
+        for (int runs = 0; runs < NUMBER_OF_TEST_RUNS; runs++) {
```

Comment:
I don't think we need repetition here. I would keep from randomizing too much when testing test methods otherwise this game never ends ;-)